cfme/tests/configure/test_session_timeout.py

```diff
@@ -35,3 +36,22 @@ def _finalize():
             "//a[normalize-space(text())='Cloud Intelligence']"
         )
     )
+
+
+@pytest.mark.tier(3)
+@pytest.mark.ignore_stream('5.8')  # Modifying settings via rest in 5.9+
+@pytest.mark.meta(blockers=[BZ(1553394)])
+def test_bind_timeout_rest(appliance, request):
+    """Sets the session timeout via REST
+        Written for BZ 1553394
```

Comment:
OPTIONAL: strange indentation, there should be a blank line between summary line and the rest of the docstring. See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
@quarckster pointed this out in another PR, I had recently re-installed and it seems I need to give pycharm settings a beatdown.